Formula/rpki-client.rb

```diff
@@ -0,0 +1,55 @@
+class RpkiClient < Formula
+  desc "OpenBSD portable rpki-client"
+  homepage "https://www.rpki-client.org/index.html"
+  url "https://ftp.openbsd.org/pub/OpenBSD/rpki-client/rpki-client-7.4.tar.gz"
+  sha256 "efd336137707ad73fa51f8480cff686c21fea7086619432069c1a02553b68d57"
+  license "ISC"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on arch: :x86_64
+  depends_on "libressl"
+  depends_on macos: :mojave
+
+  # for test later on
+  resource("testdata") do
+    # use this to limit test to one RIR's TA and the minimum set of products
+    url "https://tal.apnic.net/apnic-as0.tal"
+    sha256 "0c5c2301a1da19b20f6a5407c10c121886d6298581e8112e2f8d500ae872e69b"
+  end
+
+  def install
+    ENV.deparallelize
+
+    # --localstatedir wires things into the installed /var/{db,cache}/rpki-client paths, not the symlink farm
+    system "./configure", *std_configure_args,
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}",
+                          "--sysconfdir=#{etc}",
+                          "--localstatedir=#{var}"
+    system "make", "install"
+  end
+
+  def post_install
+    # make the var/db,cache/rpki-client dirs
+    (var/"db/rpki-client").mkpath
+    (var/"cache/rpki-client").mkpath
+
+    ohai <<~EOS
+      If you accept the ARIN Relying Party Agreement (RPA), and want to install the arin.tal:
+
+      curl -o "#{etc}/rpki/arin.tal" \
+        https://www.arin.net/resources/manage/rpki/arin.tal
+    EOS
```

Comment:
We should just make this into a more generic caveat about installing `.tal` files into `etc/"rpki"`.

Reply:
the codebase is distributed with 4 TAL from APNIC, LACNIC, AFRINIC and RIPE. Only the ARIN tal has to be installed by hand, because only ARIN require you to accept an RPA condition, and so cannot have their TAL included in the s/w distribution. All the Relying Party codebases (rcynic, rpstir, fort, routinator, rpki-client) require something like this hand install. So its not really a generic requirement.

Also, it is really clumsy doing this via <<~EOS without the caveats. When I look at other .rb, they use caveat blocks just like I did. 

I have tried ohai "..." but its truncating messages. Can you give me a complete worked example of "how to do it" because I can't find one in the Formula Cookbook, or examples on-hand in the Formula/ subdirectory

Reply:
If it's not generic then we should just drop it. We don't want to carry any documentation that isn't Homebrew specific. By "don't use caveats" I meant that this information shouldn't be in the formula.

Reply:
I've removed the ohai <<~ accordingly